pkg/pillar/cmd/zedmanager/handledomainmgr.go

```diff
@@ -158,34 +164,6 @@ func publishDomainConfig(ctx *zedmanagerContext,
 
 	key := config.Key()
 	log.Tracef("publishDomainConfig(%s)", key)
-	if ctx.hvTypeKube {
-		var imageName string
-		for _, disk := range config.DiskConfigList {
-			sub := ctx.subVolumeRefStatus
-			items := sub.GetAll()
-			for _, item := range items {
-				vrs := item.(types.VolumeRefStatus)
-				if strings.Contains(disk.VolumeKey, vrs.VolumeID.String()) {
-					if config.VirtualizationMode == types.KubeContainer {
-						imageName = vrs.ReferenceName
-					}
-					break
-				}
-			}
-			if imageName != "" {
-				break
-			}
-		}
-		config.KubeImageName = imageName
-		// set Activate to false if the config is waiting for image name
```

Comment:
remove this section, have we tried to run the native container to see if it still work?

Reply:
Yes, that's the cleanup I was talking about. I started native container multiple times. 
But I am going to amend this commit by moving to types.NOHYPER